services/ircd/anope/default.nix

```
@@ -21,7 +22,7 @@ in {
     stopIfChanged = false;
 
     serviceConfig = {
-      ExecStart = "${package}/bin/services --confdir=${configFile} --config=services.conf --dbdir=${dataDir} --localedir=/usr/lib/anope/locale --logdir=${dataDir} --modulesdir=${package}/lib/ --nofork";
+      ExecStart = "${pkg}/bin/services --confdir=${configDir} --config=${configFile} --dbdir=${dataDir} --localedir=/usr/lib/anope/locale --logdir=${dataDir} --modulesdir=${pkg}/lib/ --nofork";
       ExecReload = "${pkgs.coreutils}/bin/kill -1 $MAINPID";
       User = "anope";
```

Can you add a StateDirectory entry here to create /var/lib/anope?
```suggestion
      User = "anope";
      StateDirectory = "anope";
```